meta.yml

```diff
@@ -1,13 +1,13 @@
 fullname: Apery
-shortname: mathcomp-apery
+shortname: apery
 organization: math-comp
+opam_name: coq-mathcomp-apery
```

Comment:
We had some discussion at an earlier Mathcomp Dev [WG](https://github.com/math-comp/math-comp/wiki/Minutes-June-16-2020) about opam naming policy which was not conclusive. If I understand correctly, the hard constraint is a `coq-` prefix. Then the question is `coq-mathcomp-apery` or `coq-apery`...